delivery_multi_expense/README.rst

```diff
@@ -0,0 +1,15 @@
+Delivery multi expense
+======================
+
+This module allows you to define in sales orders, other expenses that are not
+transportation.
```

Comment:
Añade "...using delivery methods like the one defined in the module delivery."

Estaría bien también tener una sección "Usage" que explique cómo añadirlos y lo que se espera del módulo.